docs/de/docs/deployment/index.md

```diff
@@ -0,0 +1,7 @@
+# Deployment - Einführung
```

Comment:
The first one is a bit difficult. It is supposed to be translated, so I think "Bereitstellung" or "Verteilung" would be more appropriate. However, deployment is also commonly used in German.

Reply:
Yes, "Bereitstellung" is a good translation for deployment. But personally, in my opinion (native german) "Bereitstellung" seems to be more irritating than "Deployment". Maybe there is another reviewer, who want to share his opinion?